table/memtable/arena.go

```diff
@@ -76,10 +76,10 @@ func (s *arena) get(addr arenaAddr) []byte {
 
 func (s *arena) size() int64 {
 	al := s.getArenaLocator()
-	return int64(len(al.blocks) * al.blockSize)
-}
-
-func (s *arena) reset() {
+	if len(al.blocks) == 1 {
```

Comment:
If len(ai.blocks) is at least 1, we don't need the check.

Reply:
yes. I removed this unnecessary check.